packages/salesforcedx-vscode-soql/src/queryDataView/queryDataViewService.ts

```diff
@@ -113,28 +123,41 @@ export class QueryDataViewService {
     return this.currentPanel.webview;
   }
 
-  protected onDidRecieveMessageHandler(message: DataViewEvent) {
+  protected onDidRecieveMessageHandler(message: DataViewEvent): void {
     const { type, format } = message;
     switch (type) {
       case 'activate':
         this.updateWebviewWith(this.queryData);
         break;
       case 'save_records':
-        this.handleSaveRecords(format!);
+        this.handleSaveRecords(format as FileFormat);
         break;
       default:
-        console.log('unknown message type from data view');
+        const errorMessage = nls.localize('error_unknown_error', type);
+        channelService.appendLine(errorMessage);
+        trackErrorWithTelemetry('data_view_message_type', type).catch(
+          console.error
+        );
```

Comment:
Likely, this error is thrown if soql-builder-ui and soql-extension versions are not functionally in sync.  Let's hope that never happens.

<img width="1199" alt="unknown-message" src="https://user-images.githubusercontent.com/599418/101827223-3ffae700-3aed-11eb-86d3-939ce8e7d082.png">